backend/app/utils.py

```
@@ -15,31 +15,156 @@
+    """Manages temporary credential storage with encryption and TTL."""
+
+    @staticmethod
+    def _get_encryption_key():
+        """Get or create encryption key for credentials."""
+        key = getattr(settings, 'CREDENTIAL_ENCRYPTION_KEY', None)
+        if not key:
+            # Generate a key for development - in production this should be set explicitly
+            if settings.DEBUG:
+                logger.warning("CREDENTIAL_ENCRYPTION_KEY not set. Generating temporary key for development.")
+                key = Fernet.generate_key().decode()
+                setattr(settings, 'CREDENTIAL_ENCRYPTION_KEY', key)
```

Ruff (0.13.1)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)